### ISSUE TITLE
[iOS Relay][Accessibility] VoiceOver cannot navigate to “Use email mask” button from the toolbar

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#31984]
**Steps to Reproduce:**

1. Install FF and sign in.
1. Go to a Register form.
1. Enable VoiceOver.
1. In the Register modal, tap on the email field.
1. Navigate to “Use email mask”.

**Expected Result:**
VoiceOver should be able to focus and navigate to the **“Use email mask”** button.

**Actual Result:**
The **“Use email mask”** button cannot be reached when navigating the toolbar with VoiceOver.

#### Device & build information
* Device: iPhone 17 Pro Max (iOS 26.2.1)
* Build version: v9000 (67770)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-14826)

**Comment by data-sync-user** [comment]
➤ Matt Reagan commented:

I can replicate the issue but only on iOS 26. It also appears to impact other non-Relay accessory views (e.g. Use Strong Password), so it doesn’t look specific to Relay. I’m guessing the structure of the accessory view container may have changed as part of the iOS 26 update and isn’t working correctly.

On my iOS 18 device I can swipe to move between our accessory items and other toolbar/keyboard views. On iOS 26 it does not behave the same. 

+cc Winnie Teichmann Sorin Paraipan , in case they have context or it’s a known bug already.

!ios18.mov|width=320,alt="ios18.mov"!

**Comment by data-sync-user** [comment]
➤ Sorin Paraipan commented:

Thanks Matt Reagan for the context, I’ll take a look ASAP.
